routers/events.router.js

```diff
@@ -0,0 +1,84 @@
+const express = require("express");
+const router = express.Router();
+
+const { Event } = require("../models/event.model");
+
+// GET /api/events/
+router.get("/", (req, res) => {
+  const { query } = req;
+
+  Event.find(query.checkInReady === "true" ? query : undefined)
+    .populate("project")
+    .then((events) => {
+      res.json(events);
+    })
+    .catch((err) => {
+      console.log(err);
+      res.sendStatus(500).json({
+        message: `/GET Internal server error: ${err}`,
+      });
+    });
+});
+
+router.post("/", (req, res) => {
```

Comment:
Our GET query above has more validation than our POST.

Reply:
@nickbeaird Of course. See my previous comment.
@Jonathanko52 skip this comment.